relay-dynamic-config/src/defaults.rs

```diff
@@ -551,6 +551,101 @@ pub fn hardcoded_span_metrics() -> Vec<(String, Vec<MetricSpec>)> {
                             .always(), // already guarded by condition on metric
                     ],
                 },
+                MetricSpec {
+                    category: DataCategory::Span,
+                    mri: "d:transactions/webvital.score.total@ratio".into(),
+                    field: Some("span.measurements.score.total.value".into()),
```

Comment:
This is already extracted as `d:transactions/measurements.score.total@ratio` ([PR](https://github.com/getsentry/relay/pull/3154)):

https://github.com/getsentry/relay/blob/5dff9661d2bd37d77290a01581c27b563d5b632a/relay-dynamic-config/src/defaults.rs#L815-L816

@edwardgou-sentry can we get rid of `d:transactions/measurements.score.total@ratio` in favor of the new `d:transactions/webvital.score.total@ratio`? Or is the one with `measurements.` still used by the product? Asking because the `measurements.` one has caused [problems](https://github.com/getsentry/relay/pull/3324/) before.

Either way we should keep only one of the two metrics.

Reply:
Yeah I think you're right, we probably don't need two of these